crescent/commands/decorators.py

```diff
@@ -77,13 +79,21 @@ def command(
     guild: Snowflakeish | None = None,
     name: str | None = None,
     description: str | None = None,
+    default_member_permissions: UndefinedType | int | Permissions = UNDEFINED,
```

Comment:
Is there anything weird/inconsistent with using UNDEFINED here as the default? Compared to dm_enabled for example or other params. Not sure, could you explain?

Reply:
Hikari's ABC doesn't want `None` as a type so I can't use that

Reply:
Ah, that makes sense.